libs/island-ui/core/src/lib/Text/Text.tsx

```diff
@@ -85,7 +85,13 @@ export const Text = ({
       {React.Children.map<React.ReactNode, React.ReactNode>(
         children,
         (child: any) => {
-          if (typeof linkRenderer === 'function' && child?.props?.href) {
+          if (child?.type?.href && typeof linkRenderer === 'function') {
```

Comment:
@DavidBachmann might want to verify this

Reply:
This is a weird one. IIRC `child.type` is undefined outside Next (?) so I opted for the mutual `props.href` instead. Try running Storybook and see if it works.

Reply:
It should have been `child.href` not `child.type`. I have made some changes to this.